dusk-3/evm/genesis.json

```diff
@@ -0,0 +1,28 @@
+{
+  "config": {
+      "chainId": 912559,
+      "homesteadBlock": 0,
+      "eip150Block": 0,
+      "eip155Block": 0,
+      "eip158Block": 0,
+      "byzantiumBlock": 0,
+      "constantinopleBlock": 0,
+      "petersburgBlock": 0,
+      "istanbulBlock": 0,
+      "berlinBlock": 0,
+      "londonBlock": 0,
```

Comment:
Did we want to upgrade the EVM to be on the shanghai fork or w/e?

Reply:
I don't have that enabled right now.... doh.

Will sync a node locally and see if I can reach same state with it set.

Reply:
Don't have this enabled right now, can't enable from start. We can enable the fork in the future.

I missed a file update in the rush here.